app/views/admin/creator_settings/new.html.erb

```diff
@@ -1,48 +1,47 @@
-    <% if flash[:error] %>
-      <div class="crayons-notice crayons-notice--danger mb-6" role="alert">
-        <%= flash[:error] %>
-      </div>
-    <% end %>
-  </div>
+  <style>
+    <%= Rails.application.assets["setup-mode.css"].to_s.html_safe %>
+  </style>
 
-  <%= form_tag(admin_creator_settings_path, method: "post") do %>
-    <% if defined?(resource) && resource&.errors&.any? %>
-      <div class="crayons-card crayons-card--secondary crayons-notice crayons-notice--danger" role="alert" data-testid="signup-errors">
-        <div class="crayons-card__header">
-          <h1 class="crayons-card__headline">
-            Whoops, we found <%= pluralize(resource.errors.size, "problem") %>
-          </h1>
+  <main id="main-content" class="flex flex-1 justify-center flex-col crayons-layout crayons-layout--limited-xs">
+    <div aria-live="assertive">
+      <% if flash[:error] %>
+        <div class="crayons-notice crayons-notice--danger mb-6" role="alert">
+          <%= flash[:error] %>
         </div>
-        <div class="crayons-card__body">
-          <ul>
-            <% resource.errors.full_messages.each do |message| %>
-              <li><%= message %></li>
-            <% end %>
-          </ul>
+      <% end %>
+    </div>
+
+    <%= form_tag(admin_creator_settings_path, method: "post", class: "relative z-elevate p-4") do %>
+      <% if defined?(resource) && resource&.errors&.any? %>
+        <div class="crayons-card crayons-card--secondary crayons-notice crayons-notice--danger" role="alert" data-testid="signup-errors">
+          <div class="crayons-card__header">
+            <h1 class="crayons-card__headline">
+              Whoops, we found <%= pluralize(resource.errors.size, "problem") %>
+            </h1>
+          </div>
+          <div class="crayons-card__body">
+            <ul>
+              <% resource.errors.full_messages.each do |message| %>
+                <li><%= message %></li>
+              <% end %>
+            </ul>
+          </div>
         </div>
+      <% end %>
+
+      <div class="align-center">
+        <p class="pb-4 fs-3xl fw-bold">Lovely! Let's set up your Forem.</p>
+        <p class="color-base-70 fs-xl">No stress, you can always change it later.</p>
+      </div>
+      <br>
+      <%= render "form" %>
+      <div class="crayons-field mt-6 align-left">
+        <%= submit_tag "Finish", class: "crayons-btn btn--primary" %>
       </div>
     <% end %>
 
-    <div class="align-center">
-      <p class="pb-4 fs-3xl fw-bold">Lovely! Let's set up your Forem.</p>
-      <p class="color-base-70 fs-xl">No stress, you can always change it later.</p>
-    </div>
-    <br>
-  <%= render "form" %>
-  <div class="crayons-field mt-6 align-left">
-    <%= submit_tag "Finish", class: "crayons-btn btn--primary" %>
-  </div>
-  <% end %>
-<% end %>
+    <%= inline_svg_tag("forem-background.svg", aria_hidden: true, class: "forem-background absolute bottom-0 right-0 hidden m:block") %>
```

Comment:
Make the background image consistent across all setup mode pages by adding the necessary classes.

Reply:
Refactor (suggestion): Promote the forem-background.svg to a background image of the form via `background-image: image-url("forem-background.png")`. This should remove the need for a an additional element on the page, `aria-hidden`  attribute, absolutely positioning the SVG, as well as removing the need for `z-elevate` on it's sibling.

Reply:
Also, note that the `forem-background` CSS class does not appear to exist in the codebase. It can be removed.

Reply:
@nickytonline this was suggested in the initial PR and we ( I think I worked with Pawel) tried to add it as a background image however there some issues, I honestly can't remember what they were, but this was the alternative. 

I've removed `forem-background` css, thanks for spotting it!